OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -201,4 +205,20 @@ OpenSim::Object* randomize(OpenSim::Object* obj)
      return obj;
 }
 
+// Change version number of the file to 1 so that Storage can read it. This
+// function can be removed when Storage class is removed.
```

Comment:
Thank you for putting the version changing code in one place. I agree that it can be removed when our dependency on the Storage class is removed.  Please create an issue to replace the use of Storage with DataTable for comparing results of of test cases.

The comment should say that `Storage` can only read version 1 or earlier .STO files and that existing tests must revert the version number for newly created files in order to use the Storage class comparison methods.

Also renaming to:
 `revertToVersionNumber1(const std::string& currentVersionFilename,  const std::string& version1Filename)` 
would also be more informative.